packages/flutter/lib/fix_data/fix_widgets/fix_interactive_viewer.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# For details regarding the *Flutter Fix* feature, see
+# https://flutter.dev/docs/development/tools/flutter-fix
+
+# Please add new fixes to the top of the file, separated by one blank line
+# from other fixes. In a comment, include a link to the PR where the change
+# requiring the fix was made.
+
+# Every fix must be tested. See the flutter/packages/flutter/test_fixes/README.md
+# file for instructions on testing these data driven fixes.
+
+# For documentation about this file format, see
+# https://dart.dev/go/data-driven-fixes.
+
+# * Fixes in this file are for InteractiveViewer from the widgets library. *
+version: 1
+transforms:
+  # Change made in https://github.com/flutter/flutter/pull/109014
+  - title: "Migrate to 'panAxis'"
```

Comment:
I think alignPanAxis = false => PanAxis.free and alignPanAxis = true => PanAxis.aligned.
Dart fix can do this, look for examples with `oneOf:`

Reply:
Right but `alignPanAxis` is already a no-op. Converting it to the new parameter changes the current behavior (see the PR description about the other dartfix).

Reply:
It's unfortunate that we did not maintain backwards compatibility during the deprecation period and instead made a hard break going no-op, but the intention is for users to migrate to `PanAxis`. So we should absolutely provide a fix for this.